lib/exercism/guest.rb

```diff
@@ -3,6 +3,7 @@
 class Guest
   def id
   end
+  alias_method :flipper_id, :id
```

Comment:
It seems we are using this here to be sure we get `nil` for guests.  It got me confused because we want to authorize using the`github_id` but we are using `username` instead. But yeah, they are the same. 😅